scripts/train.py

```diff
@@ -0,0 +1,52 @@
+import os
+import click
+import sys
+from sklearn.externals import joblib
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+              help='Type of model to be trained')
+# @click.option('--hidden_later', '-hl', default=(15, 10),
+#               help='Hidden layers of MLP, if MLP is chosen as model_type')
+@click.option('--model_path', '-pm', type=click.Path(exists=True),
+              default=None,
+              help='Path to trained model, to be used as a base in training')
+@click.argument('dataset_dir', type=click.Path(exists=True))
```

Comment:
@rstevanak Only one last thing here. I just noticed you can pass to `click.Path` also boolean arguments `dir_okay` and `file_okay` that controls possible values of that argument. So you can probably just pass to click `dir_okay=True, file_okay=False` and it should automatically check if directory is passed to argument (http://click.pocoo.org/5/options/#tuples-as-multi-value-options)